<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1500</width>
      <height>800</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1500</width>
      <height>800</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 237, 218);</string>
    </property>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>1151</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
 
	background-color: rgb(47, 60, 126);
   border-radius:20%;
}</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);
</string>
      </property>
      <property name="text">
       <string>Sorting</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>90</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>100</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>1250</x>
       <y>60</y>
       <width>221</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
 
	background-color: rgb(47, 60, 126);
   border-radius:20%;
}</string>
     </property>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>author</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>price</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>language</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>date</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>category</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_8">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Segoe MDL2 Assets&quot;;
color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>words</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);
</string>
      </property>
      <property name="text">
       <string>Algorithum</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 184, 135);</string>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bucketsort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pigeonsort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>insertionsort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>bubblesort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>selectionsort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>merge_sort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>quicksort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>radixsort</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>countingsort</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 184, 135);</string>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>name</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>author</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>price</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>words</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>language</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>category</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonofsingle">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Single</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 184, 135);</string>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>descending</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButtonofsingle_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Multi</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>370</y>
        <width>67</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>description</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(222, 184, 135);</string>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ascending</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>descending</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>order</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>algorithm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>order</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(222, 184, 135);</string>
      </property>
      <property name="text">
       <string>attribute</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>791</width>
       <height>471</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>author</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>price</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>words</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>category</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>langguage</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>201</width>
       <height>471</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget
{
 
	background-color: rgb(47, 60, 126);
   border-radius:20%;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Scraping</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>101</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Sorting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(222, 184, 135), stop:1 rgb(240, 53, 218));
	
	
	
	color: rgb(0, 0, 0);
	border-radius:10%;
}

QPushButton:hover{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(61, 217, 245), stop:1 rgb(240, 53, 218));
	color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Searching</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2,0,36,1), stop:0.3 rgba(9,9,121,1), stop:1 rgba(0,212,255,1));
	
	
	
	color: rgb(222, 184, 135);
	border-radius:10%;
}

QPushButton:hover{
	
	color: rgb(255, 255, 255);
}background-color:rgb(37,150,190);
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
